test/Container/GeneratorFactoryTest.php

```diff
@@ -32,11 +31,11 @@ public function testInvokeCreatesGenerator()
         $this->assertInstanceOf(InjectorGenerator::class, $result);
     }
 
-    public function testFactoryUsesDiConfigContainer(): void
+    public function testFactoryUsesDiConfigContainer() : void
     {
         $container = $this->getMockBuilder(ContainerInterface::class)->getMockForAbstractClass();
         $container->method('has')->willReturnCallback(function ($type) {
-            return ($type == ConfigInterface::class);
+            return $type == ConfigInterface::class;
```

Comment:
Didn't note it previously, I would use here `===` as we started using strict types